Running schemaspy:schemaspy fails with NullPointerException

Running `mvn schemaspy:schemaspy` with maven version 3.9.8 fails with a NullPointerException


<details>
<summary>debug info, commandline and stacktrace</summary>

```
"C:\Program Files\Zulu\zulu-17\bin\java.exe" -Dmaven.multiModuleProjectDirectory=C:\Java\projects\intellij\maven\licence\database -Djansi.passthrough=true "-Dmaven.home=C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\plugins\maven\lib\maven3" "-Dclassworlds.conf=C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\plugins\maven\lib\maven3\bin\m2.conf" "-Dmaven.ext.class.path=C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\plugins\maven\lib\maven-event-listener.jar" "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\lib\idea_rt.jar=54090:C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\bin" -Dfile.encoding=UTF-8 -classpath "C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\plugins\maven\lib\maven3\boot\plexus-classworlds-2.8.0.jar;C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\plugins\maven\lib\maven3\boot\plexus-classworlds.license" org.codehaus.classworlds.Launcher -Didea.version=2024.2.4 --debug --update-snapshots -s C:\Users\anonymous\.m2\settings.xml -Dmaven.repo.local=C:\Users\anonymous\.m2\repository schemaspy:schemaspy
Apache Maven 3.9.8 (36645f6c9b5079805ea5009217e36f2cffd34256)
Maven home: C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\plugins\maven\lib\maven3
Java version: 17.0.10, vendor: Azul Systems, Inc., runtime: C:\Program Files\Zulu\zulu-17
Default locale: de_DE, platform encoding: UTF-8
OS name: "windows 11", version: "10.0", arch: "amd64", family: "windows"
[DEBUG]   Included C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\plugins\maven\lib\maven-event-listener.jar
[DEBUG] Populating class realm maven.ext
[DEBUG]   Included C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\plugins\maven\lib\maven-event-listener.jar
[DEBUG] Created new class realm maven.api
[DEBUG] Importing foreign packages into class realm maven.api
[DEBUG]   Imported: javax.annotation.* < maven.ext
[DEBUG]   Imported: javax.annotation.security.* < maven.ext
[DEBUG]   Imported: javax.inject.* < maven.ext
[DEBUG]   Imported: org.apache.maven.* < maven.ext
[DEBUG]   Imported: org.apache.maven.artifact < maven.ext
[DEBUG]   Imported: org.apache.maven.classrealm < maven.ext
[DEBUG]   Imported: org.apache.maven.cli < maven.ext
[DEBUG]   Imported: org.apache.maven.configuration < maven.ext
[DEBUG]   Imported: org.apache.maven.exception < maven.ext
[DEBUG]   Imported: org.apache.maven.execution < maven.ext
[DEBUG]   Imported: org.apache.maven.execution.scope < maven.ext
[DEBUG]   Imported: org.apache.maven.graph < maven.ext
[DEBUG]   Imported: org.apache.maven.lifecycle < maven.ext
[DEBUG]   Imported: org.apache.maven.model < maven.ext
[DEBUG]   Imported: org.apache.maven.monitor < maven.ext
[DEBUG]   Imported: org.apache.maven.plugin < maven.ext
[DEBUG]   Imported: org.apache.maven.profiles < maven.ext
[DEBUG]   Imported: org.apache.maven.project < maven.ext
[DEBUG]   Imported: org.apache.maven.reporting < maven.ext
[DEBUG]   Imported: org.apache.maven.repository < maven.ext
[DEBUG]   Imported: org.apache.maven.rtinfo < maven.ext
[DEBUG]   Imported: org.apache.maven.settings < maven.ext
[DEBUG]   Imported: org.apache.maven.toolchain < maven.ext
[DEBUG]   Imported: org.apache.maven.usability < maven.ext
[DEBUG]   Imported: org.apache.maven.wagon.* < maven.ext
[DEBUG]   Imported: org.apache.maven.wagon.authentication < maven.ext
[DEBUG]   Imported: org.apache.maven.wagon.authorization < maven.ext
[DEBUG]   Imported: org.apache.maven.wagon.events < maven.ext
[DEBUG]   Imported: org.apache.maven.wagon.observers < maven.ext
[DEBUG]   Imported: org.apache.maven.wagon.proxy < maven.ext
[DEBUG]   Imported: org.apache.maven.wagon.repository < maven.ext
[DEBUG]   Imported: org.apache.maven.wagon.resource < maven.ext
[DEBUG]   Imported: org.codehaus.classworlds < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.* < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.classworlds < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.component < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.configuration < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.container < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.context < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.lifecycle < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.logging < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.personality < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.util.xml.Xpp3Dom < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.util.xml.pull.XmlPullParser < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.util.xml.pull.XmlPullParserException < maven.ext
[DEBUG]   Imported: org.codehaus.plexus.util.xml.pull.XmlSerializer < maven.ext
[DEBUG]   Imported: org.eclipse.aether.* < maven.ext
[DEBUG]   Imported: org.eclipse.aether.artifact < maven.ext
[DEBUG]   Imported: org.eclipse.aether.collection < maven.ext
[DEBUG]   Imported: org.eclipse.aether.deployment < maven.ext
[DEBUG]   Imported: org.eclipse.aether.graph < maven.ext
[DEBUG]   Imported: org.eclipse.aether.impl < maven.ext
[DEBUG]   Imported: org.eclipse.aether.installation < maven.ext
[DEBUG]   Imported: org.eclipse.aether.internal.impl < maven.ext
[DEBUG]   Imported: org.eclipse.aether.metadata < maven.ext
[DEBUG]   Imported: org.eclipse.aether.repository < maven.ext
[DEBUG]   Imported: org.eclipse.aether.resolution < maven.ext
[DEBUG]   Imported: org.eclipse.aether.spi < maven.ext
[DEBUG]   Imported: org.eclipse.aether.transfer < maven.ext
[DEBUG]   Imported: org.eclipse.aether.util < maven.ext
[DEBUG]   Imported: org.eclipse.aether.version < maven.ext
[DEBUG]   Imported: org.fusesource.jansi.* < maven.ext
[DEBUG]   Imported: org.slf4j.* < maven.ext
[DEBUG]   Imported: org.slf4j.event.* < maven.ext
[DEBUG]   Imported: org.slf4j.helpers.* < maven.ext
[DEBUG]   Imported: org.slf4j.spi.* < maven.ext
[DEBUG] Populating class realm maven.api
[DEBUG] Created adapter factory; available factories [file-lock, rwlock-local, semaphore-local, noop]; available name mappers [discriminating, file-gav, file-hgav, file-static, gav, static]
[INFO] Error stacktraces are turned on.
[DEBUG] Message scheme: color
[DEBUG] Message styles: debug info warning error success failure strong mojo project
[DEBUG] Reading global settings from C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\plugins\maven\lib\maven3\conf\settings.xml
[DEBUG] Reading user settings from C:\Users\anonymous\.m2\settings.xml
[DEBUG] Reading global toolchains from C:\Program Files\JetBrains\IntelliJ IDEA 2024.2.4\plugins\maven\lib\maven3\conf\toolchains.xml
[DEBUG] Reading user toolchains from C:\Users\anonymous\.m2\toolchains.xml
[DEBUG] Using local repository at C:\Users\anonymous\.m2\repository
[DEBUG] Using manager EnhancedLocalRepositoryManager with priority 10.0 for C:\Users\anonymous\.m2\repository
[INFO] Scanning for projects...
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for maven-public (http://nexus.company.local:8081/repository/maven-public/).
[DEBUG] Using mirror official (https://repo.maven.apache.org/maven2/) for central (https://repo.maven.apache.org/maven2/).
[DEBUG] Using mirror archetype (http://nexus.company.local:8081/repository/maven-archetype/) for archetype (http://nexus.company.local:8081/repository/maven-archetypes/).
[DEBUG] Extension realms for project com.company.dbupdate.liquibase:ls:jar:0.0.1-SNAPSHOT: (none)
[DEBUG] Looking up lifecycle mappings for packaging jar from ClassRealm[maven.ext, parent: ClassRealm[plexus.core, parent: null]]
[DEBUG] Resolving plugin prefix schemaspy from [org.apache.maven.plugins, org.codehaus.mojo]
[DEBUG] Creating adapter using nameMapper 'gav' and factory 'rwlock-local'
[DEBUG] Using mirror official (https://repo.maven.apache.org/maven2/) for central (https://repo.maven.apache.org/maven2).
[DEBUG] Resolved plugin prefix schemaspy to nl.geodienstencentrum.maven:schemaspy-maven-plugin from POM com.company.dbupdate.liquibase:ls:jar:0.0.1-SNAPSHOT
[DEBUG] === REACTOR BUILD PLAN ================================================
[DEBUG] Project: com.company.dbupdate.liquibase:ls:jar:0.0.1-SNAPSHOT
[DEBUG] Tasks:   [schemaspy:schemaspy]
[DEBUG] Style:   Regular
[DEBUG] =======================================================================
[INFO] 
[INFO] ----------------< com.company.dbupdate.liquibase:ls >----------------
[INFO] Building ls 0.0.1-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[DEBUG] Resolving plugin prefix schemaspy from [org.apache.maven.plugins, org.codehaus.mojo]
[DEBUG] Resolved plugin prefix schemaspy to nl.geodienstencentrum.maven:schemaspy-maven-plugin from POM com.company.dbupdate.liquibase:ls:jar:0.0.1-SNAPSHOT
[DEBUG] Lifecycle clean -> [pre-clean, clean, post-clean]
[DEBUG] Lifecycle default -> [validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy]
[DEBUG] Lifecycle site -> [pre-site, site, post-site, site-deploy]
[DEBUG] === PROJECT BUILD PLAN ================================================
[DEBUG] Project:       com.company.dbupdate.liquibase:ls:0.0.1-SNAPSHOT
[DEBUG] Dependencies (collect): []
[DEBUG] Dependencies (resolve): []
[DEBUG] Repositories (dependencies): [central (http://nexus.company.local:8081/repository/maven-public/, default, releases+snapshots), official (https://repo.maven.apache.org/maven2/, default, releases+snapshots), archetype (http://nexus.company.local:8081/repository/maven-archetype/, default, snapshots)]
[DEBUG] Repositories (plugins)     : [central (http://nexus.company.local:8081/repository/maven-public/, default, releases+snapshots), official (https://repo.maven.apache.org/maven2/, default, releases)]
[DEBUG] -----------------------------------------------------------------------
[DEBUG] Goal:          nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0:schemaspy (default-cli)
[DEBUG] Style:         Regular
[DEBUG] Configuration: <?xml version="1.0" encoding="UTF-8"?>
<configuration>
  <allowHtmlInComments>${allowHtmlInComments}</allowHtmlInComments>
  <catalog default-value="%">${catalog}</catalog>
  <commentsInitiallyDisplayed>${commentsInitiallyDisplayed}</commentsInitiallyDisplayed>
  <connprops>${connprops}</connprops>
  <cssStylesheet>${cssStylesheet}</cssStylesheet>
  <database>lsdb</database>
  <databaseType>pgsql</databaseType>
  <excludeColumnNamesRegex>${excludeColumnNamesRegex}</excludeColumnNamesRegex>
  <highQuality>${highQuality}</highQuality>
  <host>localhost</host>
  <includeTableNamesRegex>${includeTableNamesRegex}</includeTableNamesRegex>
  <inputEncoding default-value="${project.build.sourceEncoding}">${encoding}</inputEncoding>
  <localRepository default-value="${localRepository}"/>
  <logLevel>${logLevel}</logLevel>
  <lowQuality>${lowQuality}</lowQuality>
  <noAds default-value="true">${noAds}</noAds>
  <noHtml>${noHtml}</noHtml>
  <noImplied>${noImplied}</noImplied>
  <noLogo default-value="true">${noLogo}</noLogo>
  <noRows>${noRows}</noRows>
  <noSchema>${noSchema}</noSchema>
  <noViews>${noViews}</noViews>
  <outputDirectory default-value="${project.build.directory}/site">schemaspy/diagram</outputDirectory>
  <outputEncoding default-value="${project.reporting.outputEncoding}">${outputEncoding}</outputEncoding>
  <password>changeit</password>
  <pathToDrivers>C:/Program Files/Java/schemaspy/postgresql-42.7.2.jar</pathToDrivers>
  <port>5432</port>
  <project default-value="${project}"/>
  <remoteRepositories default-value="${project.remoteArtifactRepositories}"/>
  <runOnExecutionRoot default-value="false">${runOnExecutionRoot}</runOnExecutionRoot>
  <schema>ls</schema>
  <schemaDescription>${schemaDescription}</schemaDescription>
  <schemas>${schemas}</schemas>
  <showAllSchemas>${showAllSchemas}</showAllSchemas>
  <singleSignOn>${singleSignOn}</singleSignOn>
  <targetDirectory default-value="${project.build.directory}">${targetDirectory}</targetDirectory>
  <useDriverManager>${useDriverManager}</useDriverManager>
  <user>postgres</user>
  <vizjs default-value="true">${vizjs}</vizjs>
</configuration>
[DEBUG] =======================================================================
[INFO] 
[INFO] --- schemaspy:5.3.0:schemaspy (default-cli) @ ls ---
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for apache.snapshots (https://repository.apache.org/snapshots).
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for apache.snapshots (http://repository.apache.org/snapshots).
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for plexus.snapshots (https://oss.sonatype.org/content/repositories/plexus-snapshots).
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for codehaus.snapshots (http://snapshots.repository.codehaus.org).
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for sonatype-nexus-snapshots (https://oss.sonatype.org/content/repositories/snapshots).
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for repository.jboss.org (http://repository.jboss.org/maven2).
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for snapshots.jboss.org (http://snapshots.jboss.org/maven2).
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for oss.sonatype.org/jboss-snapshots (http://oss.sonatype.org/content/repositories/jboss-snapshots).
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for apache.snapshots (http://people.apache.org/repo/m2-snapshot-repository).
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for snapshots (http://snapshots.maven.codehaus.org/maven2).
[DEBUG] Using mirror official (https://repo.maven.apache.org/maven2/) for central (http://repo1.maven.org/maven2).
[DEBUG] Using mirror central (http://nexus.company.local:8081/repository/maven-public/) for ow2-snapshot (http://repository.ow2.org/nexus/content/repositories/snapshots).
[DEBUG] Dependency collection stats {ConflictMarker.analyzeTime=1435500, ConflictMarker.markTime=291000, ConflictMarker.nodeCount=351, ConflictIdSorter.graphTime=954200, ConflictIdSorter.topsortTime=991100, ConflictIdSorter.conflictIdCount=135, ConflictIdSorter.conflictIdCycleCount=0, ConflictResolver.totalTime=12424500, ConflictResolver.conflictItemCount=319, DfDependencyCollector.collectTime=777498000, DfDependencyCollector.transformTime=18940400}
[DEBUG] nl.geodienstencentrum.maven:schemaspy-maven-plugin:jar:5.3.0
[DEBUG]    commons-beanutils:commons-beanutils:jar:1.9.4:compile
[DEBUG]       commons-logging:commons-logging:jar:1.2:compile
[DEBUG]       commons-collections:commons-collections:jar:3.2.2:compile
[DEBUG]    org.apache.maven.reporting:maven-reporting-impl:jar:3.2.0:compile
[DEBUG]       org.apache.maven.reporting:maven-reporting-api:jar:3.1.1:compile
[DEBUG]       org.apache.maven:maven-core:jar:3.1.0:compile
[DEBUG]          org.apache.maven:maven-model:jar:3.1.0:compile
[DEBUG]          org.apache.maven:maven-settings:jar:3.1.0:compile
[DEBUG]          org.apache.maven:maven-settings-builder:jar:3.1.0:compile
[DEBUG]          org.apache.maven:maven-repository-metadata:jar:3.1.0:compile
[DEBUG]          org.apache.maven:maven-model-builder:jar:3.1.0:compile
[DEBUG]          org.apache.maven:maven-aether-provider:jar:3.1.0:compile
[DEBUG]             org.eclipse.aether:aether-spi:jar:0.9.0.M2:compile
[DEBUG]          org.eclipse.aether:aether-impl:jar:0.9.0.M2:compile
[DEBUG]          org.eclipse.aether:aether-api:jar:0.9.0.M2:compile
[DEBUG]          org.eclipse.aether:aether-util:jar:0.9.0.M2:compile
[DEBUG]          org.eclipse.sisu:org.eclipse.sisu.plexus:jar:0.0.0.M2a:compile
[DEBUG]             javax.enterprise:cdi-api:jar:1.0:compile
[DEBUG]                javax.annotation:jsr250-api:jar:1.0:compile
[DEBUG]                javax.inject:javax.inject:jar:1:compile
[DEBUG]             com.google.guava:guava:jar:10.0.1:compile
[DEBUG]                com.google.code.findbugs:jsr305:jar:1.3.9:compile
[DEBUG]             org.sonatype.sisu:sisu-guice:jar:no_aop:3.1.0:compile
[DEBUG]                aopalliance:aopalliance:jar:1.0:compile
[DEBUG]             org.eclipse.sisu:org.eclipse.sisu.inject:jar:0.0.0.M2a:compile
[DEBUG]                asm:asm:jar:3.3.1:compile
[DEBUG]          org.codehaus.plexus:plexus-interpolation:jar:1.16:compile
[DEBUG]          org.codehaus.plexus:plexus-classworlds:jar:2.4.2:compile
[DEBUG]          org.codehaus.plexus:plexus-component-annotations:jar:1.5.5:compile
[DEBUG]          org.sonatype.plexus:plexus-sec-dispatcher:jar:1.3:compile
[DEBUG]             org.sonatype.plexus:plexus-cipher:jar:1.4:compile
[DEBUG]       org.apache.maven:maven-artifact:jar:3.1.0:compile
[DEBUG]       org.apache.maven:maven-plugin-api:jar:3.1.0:compile
[DEBUG]       org.apache.maven.shared:maven-shared-utils:jar:3.3.4:compile
[DEBUG]       org.apache.maven.doxia:doxia-sink-api:jar:1.11.1:compile
[DEBUG]          org.apache.maven.doxia:doxia-logging-api:jar:1.11.1:compile
[DEBUG]       org.apache.maven.doxia:doxia-decoration-model:jar:1.11.1:compile
[DEBUG]       org.apache.maven.doxia:doxia-core:jar:1.11.1:compile
[DEBUG]          org.codehaus.plexus:plexus-container-default:jar:2.1.0:compile
[DEBUG]             org.apache.xbean:xbean-reflect:jar:3.7:compile
[DEBUG]             com.google.collections:google-collections:jar:1.0:compile
[DEBUG]          org.apache.commons:commons-lang3:jar:3.8.1:compile
[DEBUG]          org.apache.commons:commons-text:jar:1.3:compile
[DEBUG]          org.apache.httpcomponents:httpclient:jar:4.5.13:compile
[DEBUG]             commons-codec:commons-codec:jar:1.11:compile
[DEBUG]          org.apache.httpcomponents:httpcore:jar:4.4.14:compile
[DEBUG]       org.apache.maven.doxia:doxia-integration-tools:jar:1.11.1:compile
[DEBUG]          org.codehaus.plexus:plexus-i18n:jar:1.0-beta-10:compile
[DEBUG]       org.apache.maven.doxia:doxia-site-renderer:jar:1.11.1:compile
[DEBUG]          org.apache.maven.doxia:doxia-skin-model:jar:1.11.1:compile
[DEBUG]          org.apache.maven.doxia:doxia-module-xhtml:jar:1.11.1:compile
[DEBUG]          org.apache.maven.doxia:doxia-module-xhtml5:jar:1.11.1:compile
[DEBUG]          org.codehaus.plexus:plexus-velocity:jar:1.2:compile
[DEBUG]          org.apache.velocity:velocity:jar:1.7:compile
[DEBUG]             commons-lang:commons-lang:jar:2.4:compile
[DEBUG]          org.apache.velocity:velocity-tools:jar:2.0:compile
[DEBUG]             commons-digester:commons-digester:jar:1.8:compile
[DEBUG]             commons-chain:commons-chain:jar:1.1:compile
[DEBUG]             dom4j:dom4j:jar:1.1:compile
[DEBUG]             oro:oro:jar:2.0.8:compile
[DEBUG]       org.codehaus.plexus:plexus-utils:jar:3.3.1:compile
[DEBUG]    org.schemaspy:schemaspy:jar:6.2.4:compile
[DEBUG]       com.github.spullara.mustache.java:compiler:jar:0.9.10:compile
[DEBUG]       commons-io:commons-io:jar:2.7:compile
[DEBUG]       com.vladsch.flexmark:flexmark:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-util:jar:0.34.32:compile
[DEBUG]       com.vladsch.flexmark:flexmark-profile-pegdown:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-abbreviation:jar:0.34.32:compile
[DEBUG]             com.vladsch.flexmark:flexmark-formatter:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-anchorlink:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-aside:jar:0.34.32:compile
[DEBUG]             com.vladsch.flexmark:flexmark-jira-converter:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-autolink:jar:0.34.32:compile
[DEBUG]             org.nibor.autolink:autolink:jar:0.6.0:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-definition:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-emoji:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-escaped-character:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-footnotes:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-gfm-strikethrough:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-gfm-tasklist:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-ins:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-jekyll-front-matter:jar:0.34.32:compile
[DEBUG]             com.vladsch.flexmark:flexmark-ext-yaml-front-matter:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-superscript:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-tables:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-toc:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-typographic:jar:0.34.32:compile
[DEBUG]          com.vladsch.flexmark:flexmark-ext-wikilink:jar:0.34.32:compile
[DEBUG]       org.json:json:jar:20230227:compile
[DEBUG]       org.springframework.boot:spring-boot-starter:jar:2.2.11.RELEASE:compile
[DEBUG]          org.springframework.boot:spring-boot-starter-logging:jar:2.2.11.RELEASE:compile
[DEBUG]             ch.qos.logback:logback-classic:jar:1.2.3:compile
[DEBUG]                ch.qos.logback:logback-core:jar:1.2.3:compile
[DEBUG]                org.slf4j:slf4j-api:jar:1.7.25:compile
[DEBUG]             org.apache.logging.log4j:log4j-to-slf4j:jar:2.12.1:compile
[DEBUG]                org.apache.logging.log4j:log4j-api:jar:2.12.1:compile
[DEBUG]             org.slf4j:jul-to-slf4j:jar:1.7.30:compile
[DEBUG]          jakarta.annotation:jakarta.annotation-api:jar:1.3.5:compile
[DEBUG]          org.yaml:snakeyaml:jar:1.25:runtime
[DEBUG]       org.springframework.boot:spring-boot:jar:2.2.11.RELEASE:compile
[DEBUG]       org.springframework.boot:spring-boot-autoconfigure:jar:2.2.11.RELEASE:compile
[DEBUG]       org.springframework:spring-core:jar:5.2.10.RELEASE:compile
[DEBUG]          org.springframework:spring-jcl:jar:5.2.10.RELEASE:compile
[DEBUG]       org.springframework:spring-context:jar:5.2.10.RELEASE:compile
[DEBUG]          org.springframework:spring-aop:jar:5.2.10.RELEASE:compile
[DEBUG]          org.springframework:spring-expression:jar:5.2.10.RELEASE:compile
[DEBUG]       org.springframework:spring-beans:jar:5.2.10.RELEASE:compile
[DEBUG]       org.hamcrest:hamcrest-core:jar:2.1:compile
[DEBUG]          org.hamcrest:hamcrest:jar:2.1:compile
[DEBUG]       com.beust:jcommander:jar:1.69:compile
[DEBUG]       xalan:xalan:jar:2.7.2:compile
[DEBUG]          xalan:serializer:jar:2.7.2:compile
[DEBUG]             xml-apis:xml-apis:jar:1.3.04:compile
[DEBUG]       org.openjdk.nashorn:nashorn-core:jar:15.4:compile
[DEBUG]          org.ow2.asm:asm:jar:7.3.1:compile
[DEBUG]          org.ow2.asm:asm-commons:jar:7.3.1:compile
[DEBUG]             org.ow2.asm:asm-analysis:jar:7.3.1:compile
[DEBUG]          org.ow2.asm:asm-tree:jar:7.3.1:compile
[DEBUG]          org.ow2.asm:asm-util:jar:7.3.1:compile
[DEBUG]    org.apache.derby:derby:jar:10.15.2.0:compile (optional)
[DEBUG]       org.apache.derby:derbyshared:jar:10.15.2.0:compile (optional)
[DEBUG]    org.apache.derby:derbytools:jar:10.15.2.0:compile (optional)
[DEBUG]    org.hsqldb:hsqldb:jar:2.7.3:compile (optional)
[DEBUG]    org.postgresql:postgresql:jar:42.7.4:compile (optional)
[DEBUG]       org.checkerframework:checker-qual:jar:3.42.0:runtime (optional)
[DEBUG]    com.microsoft.sqlserver:mssql-jdbc:jar:12.8.1.jre11:compile (optional)
[DEBUG]    com.mysql:mysql-connector-j:jar:9.0.0:compile (optional)
[DEBUG]       com.google.protobuf:protobuf-java:jar:4.26.1:compile (optional)
[DEBUG]    com.oracle.database.jdbc:ojdbc11:jar:23.5.0.24.07:compile (optional)
[DEBUG] Created new class realm plugin>nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0
[DEBUG] Importing foreign packages into class realm plugin>nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0
[DEBUG]   Imported:  < maven.api
[DEBUG] Populating class realm plugin>nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0
[DEBUG]   Included: nl.geodienstencentrum.maven:schemaspy-maven-plugin:jar:5.3.0
[DEBUG]   Included: commons-beanutils:commons-beanutils:jar:1.9.4
[DEBUG]   Included: commons-logging:commons-logging:jar:1.2
[DEBUG]   Included: commons-collections:commons-collections:jar:3.2.2
[DEBUG]   Included: org.apache.maven.reporting:maven-reporting-impl:jar:3.2.0
[DEBUG]   Included: org.apache.maven.reporting:maven-reporting-api:jar:3.1.1
[DEBUG]   Included: javax.enterprise:cdi-api:jar:1.0
[DEBUG]   Included: javax.annotation:jsr250-api:jar:1.0
[DEBUG]   Included: com.google.guava:guava:jar:10.0.1
[DEBUG]   Included: com.google.code.findbugs:jsr305:jar:1.3.9
[DEBUG]   Included: org.sonatype.sisu:sisu-guice:jar:no_aop:3.1.0
[DEBUG]   Included: aopalliance:aopalliance:jar:1.0
[DEBUG]   Included: org.eclipse.sisu:org.eclipse.sisu.inject:jar:0.0.0.M2a
[DEBUG]   Included: asm:asm:jar:3.3.1
[DEBUG]   Included: org.codehaus.plexus:plexus-interpolation:jar:1.16
[DEBUG]   Included: org.codehaus.plexus:plexus-component-annotations:jar:1.5.5
[DEBUG]   Included: org.sonatype.plexus:plexus-sec-dispatcher:jar:1.3
[DEBUG]   Included: org.sonatype.plexus:plexus-cipher:jar:1.4
[DEBUG]   Included: org.apache.maven.shared:maven-shared-utils:jar:3.3.4
[DEBUG]   Included: org.apache.maven.doxia:doxia-sink-api:jar:1.11.1
[DEBUG]   Included: org.apache.maven.doxia:doxia-logging-api:jar:1.11.1
[DEBUG]   Included: org.apache.maven.doxia:doxia-decoration-model:jar:1.11.1
[DEBUG]   Included: org.apache.maven.doxia:doxia-core:jar:1.11.1
[DEBUG]   Included: org.apache.xbean:xbean-reflect:jar:3.7
[DEBUG]   Included: com.google.collections:google-collections:jar:1.0
[DEBUG]   Included: org.apache.commons:commons-lang3:jar:3.8.1
[DEBUG]   Included: org.apache.commons:commons-text:jar:1.3
[DEBUG]   Included: org.apache.httpcomponents:httpclient:jar:4.5.13
[DEBUG]   Included: commons-codec:commons-codec:jar:1.11
[DEBUG]   Included: org.apache.httpcomponents:httpcore:jar:4.4.14
[DEBUG]   Included: org.apache.maven.doxia:doxia-integration-tools:jar:1.11.1
[DEBUG]   Included: org.codehaus.plexus:plexus-i18n:jar:1.0-beta-10
[DEBUG]   Included: org.apache.maven.doxia:doxia-site-renderer:jar:1.11.1
[DEBUG]   Included: org.apache.maven.doxia:doxia-skin-model:jar:1.11.1
[DEBUG]   Included: org.apache.maven.doxia:doxia-module-xhtml:jar:1.11.1
[DEBUG]   Included: org.apache.maven.doxia:doxia-module-xhtml5:jar:1.11.1
[DEBUG]   Included: org.codehaus.plexus:plexus-velocity:jar:1.2
[DEBUG]   Included: org.apache.velocity:velocity:jar:1.7
[DEBUG]   Included: commons-lang:commons-lang:jar:2.4
[DEBUG]   Included: org.apache.velocity:velocity-tools:jar:2.0
[DEBUG]   Included: commons-digester:commons-digester:jar:1.8
[DEBUG]   Included: commons-chain:commons-chain:jar:1.1
[DEBUG]   Included: dom4j:dom4j:jar:1.1
[DEBUG]   Included: oro:oro:jar:2.0.8
[DEBUG]   Included: org.codehaus.plexus:plexus-utils:jar:3.3.1
[DEBUG]   Included: org.schemaspy:schemaspy:jar:6.2.4
[DEBUG]   Included: com.github.spullara.mustache.java:compiler:jar:0.9.10
[DEBUG]   Included: commons-io:commons-io:jar:2.7
[DEBUG]   Included: com.vladsch.flexmark:flexmark:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-util:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-profile-pegdown:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-abbreviation:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-formatter:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-anchorlink:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-aside:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-jira-converter:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-autolink:jar:0.34.32
[DEBUG]   Included: org.nibor.autolink:autolink:jar:0.6.0
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-definition:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-emoji:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-escaped-character:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-footnotes:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-gfm-strikethrough:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-gfm-tasklist:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-ins:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-jekyll-front-matter:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-yaml-front-matter:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-superscript:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-tables:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-toc:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-typographic:jar:0.34.32
[DEBUG]   Included: com.vladsch.flexmark:flexmark-ext-wikilink:jar:0.34.32
[DEBUG]   Included: org.json:json:jar:20230227
[DEBUG]   Included: org.springframework.boot:spring-boot-starter:jar:2.2.11.RELEASE
[DEBUG]   Included: org.springframework.boot:spring-boot-starter-logging:jar:2.2.11.RELEASE
[DEBUG]   Included: ch.qos.logback:logback-classic:jar:1.2.3
[DEBUG]   Included: ch.qos.logback:logback-core:jar:1.2.3
[DEBUG]   Included: org.apache.logging.log4j:log4j-to-slf4j:jar:2.12.1
[DEBUG]   Included: org.apache.logging.log4j:log4j-api:jar:2.12.1
[DEBUG]   Included: org.slf4j:jul-to-slf4j:jar:1.7.30
[DEBUG]   Included: jakarta.annotation:jakarta.annotation-api:jar:1.3.5
[DEBUG]   Included: org.yaml:snakeyaml:jar:1.25
[DEBUG]   Included: org.springframework.boot:spring-boot:jar:2.2.11.RELEASE
[DEBUG]   Included: org.springframework.boot:spring-boot-autoconfigure:jar:2.2.11.RELEASE
[DEBUG]   Included: org.springframework:spring-core:jar:5.2.10.RELEASE
[DEBUG]   Included: org.springframework:spring-jcl:jar:5.2.10.RELEASE
[DEBUG]   Included: org.springframework:spring-context:jar:5.2.10.RELEASE
[DEBUG]   Included: org.springframework:spring-aop:jar:5.2.10.RELEASE
[DEBUG]   Included: org.springframework:spring-expression:jar:5.2.10.RELEASE
[DEBUG]   Included: org.springframework:spring-beans:jar:5.2.10.RELEASE
[DEBUG]   Included: org.hamcrest:hamcrest-core:jar:2.1
[DEBUG]   Included: org.hamcrest:hamcrest:jar:2.1
[DEBUG]   Included: com.beust:jcommander:jar:1.69
[DEBUG]   Included: xalan:xalan:jar:2.7.2
[DEBUG]   Included: xalan:serializer:jar:2.7.2
[DEBUG]   Included: xml-apis:xml-apis:jar:1.3.04
[DEBUG]   Included: org.openjdk.nashorn:nashorn-core:jar:15.4
[DEBUG]   Included: org.ow2.asm:asm:jar:7.3.1
[DEBUG]   Included: org.ow2.asm:asm-commons:jar:7.3.1
[DEBUG]   Included: org.ow2.asm:asm-analysis:jar:7.3.1
[DEBUG]   Included: org.ow2.asm:asm-tree:jar:7.3.1
[DEBUG]   Included: org.ow2.asm:asm-util:jar:7.3.1
[DEBUG]   Included: org.apache.derby:derby:jar:10.15.2.0
[DEBUG]   Included: org.apache.derby:derbyshared:jar:10.15.2.0
[DEBUG]   Included: org.apache.derby:derbytools:jar:10.15.2.0
[DEBUG]   Included: org.hsqldb:hsqldb:jar:2.7.3
[DEBUG]   Included: org.postgresql:postgresql:jar:42.7.4
[DEBUG]   Included: org.checkerframework:checker-qual:jar:3.42.0
[DEBUG]   Included: com.microsoft.sqlserver:mssql-jdbc:jar:12.8.1.jre11
[DEBUG]   Included: com.mysql:mysql-connector-j:jar:9.0.0
[DEBUG]   Included: com.google.protobuf:protobuf-java:jar:4.26.1
[DEBUG]   Included: com.oracle.database.jdbc:ojdbc11:jar:23.5.0.24.07
[DEBUG] Loading mojo nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0:schemaspy from plugin realm ClassRealm[plugin>nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0, parent: jdk.internal.loader.ClassLoaders$AppClassLoader@60e53b93]
[DEBUG] Setting property: velocimacro.permissions.allow.inline.to.replace.global => 'true'.
[DEBUG] Setting property: eventhandler.include.class => 'org.apache.velocity.app.event.implement.IncludeRelativePath'.
[DEBUG] Setting property: velocimacro.messages.on => 'false'.
[DEBUG] Setting property: file.resource.loader.path => ''.
[DEBUG] Setting property: resource.loader => 'classpath,file'.
[DEBUG] Setting property: file.resource.loader.class => 'org.apache.velocity.runtime.resource.loader.FileResourceLoader'.
[DEBUG] Setting property: runtime.log.invalid.references => 'false'.
[DEBUG] Setting property: resource.manager.logwhenfound => 'false'.
[DEBUG] Setting property: classpath.resource.loader.class => 'org.apache.velocity.runtime.resource.loader.ClasspathResourceLoader'.
[DEBUG] Initializing Velocity, Calling init()...
[DEBUG] *******************************************************************
[DEBUG] Starting Apache Velocity v1.7 (compiled: 2010-11-19 12:14:37)
[DEBUG] RuntimeInstance initializing.
[DEBUG] Default Properties File: org\apache\velocity\runtime\defaults\velocity.properties
[DEBUG] Default ResourceManager initializing. (class org.apache.velocity.runtime.resource.ResourceManagerImpl)
[DEBUG] ResourceLoader instantiated: org.apache.velocity.runtime.resource.loader.ClasspathResourceLoader
[DEBUG] ClasspathResourceLoader : initialization complete.
[DEBUG] ResourceLoader instantiated: org.apache.velocity.runtime.resource.loader.FileResourceLoader
[DEBUG] FileResourceLoader : initialization starting.
[DEBUG] Do unicode file recognition:  false
[DEBUG] FileResourceLoader : adding path ''
[DEBUG] FileResourceLoader : initialization complete.
[DEBUG] ResourceCache: initialized (class org.apache.velocity.runtime.resource.ResourceCacheImpl) with class java.util.Collections$SynchronizedMap cache map.
[DEBUG] Default ResourceManager initialization complete.
[DEBUG] Loaded System Directive: org.apache.velocity.runtime.directive.Stop
[DEBUG] Loaded System Directive: org.apache.velocity.runtime.directive.Define
[DEBUG] Loaded System Directive: org.apache.velocity.runtime.directive.Break
[DEBUG] Loaded System Directive: org.apache.velocity.runtime.directive.Evaluate
[DEBUG] Loaded System Directive: org.apache.velocity.runtime.directive.Literal
[DEBUG] Loaded System Directive: org.apache.velocity.runtime.directive.Macro
[DEBUG] Loaded System Directive: org.apache.velocity.runtime.directive.Parse
[DEBUG] Loaded System Directive: org.apache.velocity.runtime.directive.Include
[DEBUG] Loaded System Directive: org.apache.velocity.runtime.directive.Foreach
[DEBUG] Created '20' parsers.
[DEBUG] Velocimacro : initialization starting.
[DEBUG] Velocimacro : allowInline = true : VMs can be defined inline in templates
[DEBUG] Velocimacro : allowInlineToOverride = true : VMs defined inline may replace previous VM definitions
[DEBUG] Velocimacro : allowInlineLocal = false : VMs defined inline will be global in scope if allowed.
[DEBUG] Velocimacro : autoload off : VM system will not automatically reload global library macros
[DEBUG] Velocimacro : Velocimacro : initialization complete.
[DEBUG] RuntimeInstance successfully initialized.
[DEBUG] Configuring mojo execution 'nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0:schemaspy:default-cli' with basic configurator -->
[DEBUG]   (f) catalog = %
[DEBUG]   (f) database = lsdb
[DEBUG]   (f) databaseType = pgsql
[DEBUG]   (f) host = localhost
[DEBUG]   (f) localRepository =       id: local
      url: file:///C:/Users/anonymous/.m2/repository/
   layout: default
snapshots: [enabled => true, update => always]
 releases: [enabled => true, update => always]
   blocked: false

[DEBUG]   (f) noAds = true
[DEBUG]   (f) noLogo = true
[DEBUG]   (f) outputDirectory = schemaspy/diagram
[DEBUG]   (f) password = changeit
[DEBUG]   (f) pathToDrivers = C:/Program Files/Java/schemaspy/postgresql-42.7.2.jar
[DEBUG]   (f) port = 5432
[DEBUG]   (f) project = MavenProject: com.company.dbupdate.liquibase:ls:0.0.1-SNAPSHOT @ C:\Java\projects\intellij\maven\licence\database\pom.xml
[DEBUG]   (f) remoteRepositories = [      id: central
      url: http://nexus.company.local:8081/repository/maven-public/
   layout: default
snapshots: [enabled => true, update => always]
 releases: [enabled => true, update => daily]
   blocked: false
,       id: official
      url: https://repo.maven.apache.org/maven2/
   layout: default
    proxy: 192.168.9.2:3128
snapshots: [enabled => true, update => daily]
 releases: [enabled => true, update => daily]
   blocked: false
,       id: archetype
      url: http://nexus.company.local:8081/repository/maven-archetype/
   layout: default
snapshots: [enabled => true, update => daily]
 releases: [enabled => false, update => daily]
   blocked: false
]
[DEBUG]   (f) runOnExecutionRoot = false
[DEBUG]   (f) schema = ls
[DEBUG]   (f) targetDirectory = C:\Java\projects\intellij\maven\licence\database\target
[DEBUG]   (f) user = postgres
[DEBUG]   (f) vizjs = true
[DEBUG] -- end configuration --
[INFO] Rendering content with org.apache.maven.skins:maven-default-skin:jar:1.3 skin.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.487 s
[INFO] Finished at: 2025-01-15T10:06:44+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0:schemaspy (default-cli) on project ls: Execution default-cli of goal nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0:schemaspy failed: Cannot invoke "org.apache.maven.project.MavenProject.getName()" because "this.project" is null -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0:schemaspy (default-cli) on project ls: Execution default-cli of goal nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0:schemaspy failed: Cannot invoke "org.apache.maven.project.MavenProject.getName()" because "this.project" is null
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:333)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:903)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:280)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:203)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at org.codehaus.classworlds.Launcher.main (Launcher.java:41)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default-cli of goal nl.geodienstencentrum.maven:schemaspy-maven-plugin:5.3.0:schemaspy failed: Cannot invoke "org.apache.maven.project.MavenProject.getName()" because "this.project" is null
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:133)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:903)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:280)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:203)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at org.codehaus.classworlds.Launcher.main (Launcher.java:41)
Caused by: java.lang.NullPointerException: Cannot invoke "org.apache.maven.project.MavenProject.getName()" because "this.project" is null
    at org.apache.maven.reporting.AbstractMavenReport.createSiteRenderingContext (AbstractMavenReport.java:218)
    at org.apache.maven.reporting.AbstractMavenReport.execute (AbstractMavenReport.java:156)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:126)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:903)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:280)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:203)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at org.codehaus.classworlds.Launcher.main (Launcher.java:41)
[ERROR] 
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[DEBUG] Shutting down adapter factory; available factories [file-lock, rwlock-local, semaphore-local, noop]; available name mappers [discriminating, file-gav, file-hgav, file-static, gav, static]
[DEBUG] Shutting down 'file-lock' factory
[DEBUG] Shutting down 'rwlock-local' factory
[DEBUG] Shutting down 'semaphore-local' factory
[DEBUG] Shutting down 'noop' factory

Process finished with exit code 1
```
</details>

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -11,9 +11,6 @@ on:
 
 env:
   MAVEN_OPTS: -Djava.awt.headless=true
-  PGPASSWORD: postgres
-  PGUSER: postgres
-  PGDATABASE: postgres
 
 jobs:
   build:
@@ -43,7 +40,7 @@ jobs:
 
       - name: 'Startup databases'
         run: |
-          build/ci/start-pgsql.sh 16-3.4
+          build/ci/start-pgsql.sh 17-3.5
           build/ci/start-oracle.sh 21.3.0
           build/ci/start-mysql.sh 8
           build/ci/start-mssql.sh 2022-latest
@@ -56,8 +53,8 @@ jobs:
 
       - name: Setup databases
         run: |
-          psql -U postgres -h localhost -c 'CREATE DATABASE test;'
-          psql -U postgres -h localhost -d test -f ./src/test/resources/sql/pgsql.sql
+          PGPASSWORD=postgres psql -U postgres -h localhost -c 'CREATE DATABASE test;'
+          PGPASSWORD=postgres psql -U postgres -h localhost -d test -f ./src/test/resources/sql/pgsql.sql
           docker exec -i mysql1 mysql -uroot -pmysql -e 'CREATE DATABASE IF NOT EXISTS test;'
           docker exec -i mysql1 mysql -uroot -pmysql test < ./src/test/resources/sql/mysql.sql
           docker exec -i sql1 /opt/mssql-tools18/bin/sqlcmd -S localhost -U sa -P 'Password12!' -No -d 'master' -Q 'CREATE DATABASE test'
@@ -67,7 +64,7 @@ jobs:
 
       - name: Test
         run: |
-          mvn -e -B -fae -Pmaven-ci clean test -Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss,SSS -Dfailsafe.useFile=false -Djava.awt.headless=true
+          mvn -e -B -fae -Pmaven-ci clean verify -Dorg.slf4j.simpleLogger.showDateTime=true -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss,SSS -Dfailsafe.useFile=false -Djava.awt.headless=true
           mvn -e -B -fae -Pmaven-ci jacoco:report coveralls:report -q -Dlicense.skip=true -DrepoToken=$GITHUB_TOKEN -DserviceName=github -Djava.awt.headless=true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

--- a/pom.xml
+++ b/pom.xml
@@ -318,6 +318,10 @@
                     <version>${maven-surefire-plugin.version}</version>
                 </plugin>
                 <plugin>
+                    <artifactId>maven-failsafe-plugin</artifactId>
+                    <version>${maven-surefire-plugin.version}</version>
+                </plugin>
+                <plugin>
                     <groupId>org.jacoco</groupId>
                     <artifactId>jacoco-maven-plugin</artifactId>
                     <version>${jacoco-maven-plugin.version}</version>
@@ -435,8 +439,54 @@
                     <argLine>${surefireArgLine}</argLine>
                     <excludes>
                         <exclude>${surefire.excludes.files}</exclude>
+                        <exclude>**/*IntegrationTest.java</exclude>
                     </excludes>
                 </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-jar</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <mkdir dir="${project.build.directory}/testRepo/nl/geodienstencentrum/maven/${project.artifactId}/${project.version}"/>
+                                <copy file="${project.build.directory}/${project.build.finalName}.jar"
+                                      tofile="${project.build.directory}/testRepo/nl/geodienstencentrum/maven/${project.artifactId}/${project.version}/${project.build.finalName}.jar"
+                                      verbose="true" overwrite="true"/>
+                                <copy file="${project.basedir}/pom.xml"
+                                      tofile="${project.build.directory}/testRepo/nl/geodienstencentrum/maven/${project.artifactId}/${project.version}/${project.build.finalName}.pom"
+                                      verbose="true" overwrite="true"/>
+                            </target>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <trimStackTrace>false</trimStackTrace>
+                    <includes>
+                        <include>**/*IntegrationTest.java</include>
+                    </includes>
+                    <!--suppress UnresolvedMavenProperty -->
+                    <argLine>${failsafeArgLine}</argLine>
+                    <environmentVariables>
+                        <PLUGIN_VERSION>${project.version}</PLUGIN_VERSION>
+                    </environmentVariables>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>integration-test</goal>
+                            <goal>verify</goal>
+                        </goals>
+                    </execution>
+                </executions>
             </plugin>
             <plugin>
                 <groupId>org.jacoco</groupId>

--- a/src/main/java/com/wakaleo/schemaspy/SchemaSpyReport.java
+++ b/src/main/java/com/wakaleo/schemaspy/SchemaSpyReport.java
@@ -1,513 +1,401 @@
 package com.wakaleo.schemaspy;
 
-import org.apache.maven.doxia.siterenderer.Renderer;
-import org.apache.maven.plugins.annotations.Component;
-import org.apache.maven.plugins.annotations.LifecyclePhase;
-import org.apache.maven.plugins.annotations.Mojo;
-import org.apache.maven.plugins.annotations.Parameter;
-import org.apache.maven.project.MavenProject;
-import org.apache.maven.reporting.AbstractMavenReport;
-import org.apache.maven.reporting.MavenReportException;
-
 import java.io.File;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Locale;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.reporting.AbstractMavenReport;
+import org.apache.maven.reporting.MavenReportException;
 
 /**
  * The SchemaSpy Maven plugin report.
  *
+ * <p>This plugin is designed to generate SchemaSpy report for a Maven website. SchemaSpy also may
+ * need the Graphviz tool (https://www.graphviz.org/) in order to generate graphical representations
+ * of the table/view relationships, so this needs to be installed on your machine.
+ *
+ * <p>The schemaspy goal invokes the SchemaSpy command-line tool. SchemaSpy generates a graphical
+ * and HTML report describing a given relational database.
+ *
  * @author John Smart
- *           The SchemaSpy Maven plugin This plugin is designed to generate
- *           SchemaSpy report for a Maven web site.
- *           SchemaSpy also may need the Graphviz tool (https://www.graphviz.org/) in order to
- *           generate graphical representations of the table/view relationships,
- *           so this needs to be installed on your machine.
- *           <p/>
- *           The schemaspy goal invokes the SchemaSpy command-line tool.
- *           SchemaSpy generates a graphical and HTML report describing a given
- *           relational database.
+ * @author mprins
  */
 @Mojo(name = "schemaspy", defaultPhase = LifecyclePhase.SITE)
 public class SchemaSpyReport extends AbstractMavenReport {
 
-    /**
-     * The output directory for the intermediate report.
-     *
-     */
-    @Parameter(property = "targetDirectory", defaultValue = "${project.build.directory}")
-    private File targetDirectory;
+  @Parameter(property = "vizjs", defaultValue = "true")
+  protected boolean vizjs = false;
 
-    /**
-     * The output directory where the final HTML reports will be generated. Note
-     * that the reports are always generated in a directory called "schemaspy".
-     * The output directory refers to the directory in which the "schemaspy"
-     * will be generated.
-     *
-     */
-    @Parameter(property = "outputDirectory", defaultValue = "${project.build.directory}/site")
-    private String outputDirectory;
+  /**
+   * Whether to create the report only on the execution root of a multi-module project.
+   *
+   * @since 5.0.4
+   */
+  @Parameter(property = "runOnExecutionRoot", defaultValue = "false")
+  protected boolean runOnExecutionRoot = false;
 
-    /**
-     * Site rendering component for generating the HTML report.
-     */
-    @Component
-    private Renderer siteRenderer;
+  /** The output directory for the intermediate report. */
+  @Parameter(property = "targetDirectory", defaultValue = "${project.build.directory}")
+  private File targetDirectory;
 
-    /**
-     * The Maven project object.
-     */
-    @Parameter(defaultValue = "${project}", readonly = true, required = true)
-    private MavenProject project;
+  /** The name of the database being analysed. */
+  @Parameter(property = "database", required = true)
+  private String database;
 
+  /** The host address of the database being analysed. */
+  @Parameter(property = "host")
+  private String host;
 
-    /**
-     * The name of the database being analysed.
-     */
-    @Parameter (property = "database", required = true)
-    private String database;
+  /** The port, required by some drivers. */
+  @Parameter(property = "port")
+  private String port;
 
-    /**
-     * The host address of the database being analysed.
-     */
-    @Parameter(property = "host")
-    private String host;
+  /** The type of database being analysed — defaults to ora. */
+  @Parameter(property = "databaseType")
+  private String databaseType;
 
-    /**
-     * The port, required by some drivers.
-     */
-    @Parameter(property = "port")
-    private String port;
+  /** Connect to the database with this user id. */
+  @Parameter(property = "user")
+  private String user;
 
-//    /**
-//     * The JDBC URL to be used to connect to the database. Rather than defining
-//     * the database and host names and letting SchemaSpy build the URL, you can
-//     * alternatively specify the complete JDBC URL using this parameter. If this
-//     * parameter is defined, it will override the host address (which, as a
-//     * result, is not needed). Note that you still need to specify the database
-//     * type, since SchemaSpy uses its own database properties file for extra
-//     * information about each database.
-//     *
-//     * @todo Would it be possible to guess the database type from the form of
-//     *       the URL?
-//     */
-//    @Parameter
-//    private String jdbcUrl;
+  /** Database schema to use — defaults to the specified user. */
+  @Parameter(property = "schema")
+  private String schema;
 
-    /**
-     * The type of database being analysed - defaults to ora.
-     */
-    @Parameter (property = "databaseType")
-    private String databaseType;
+  /** Database password to use — defaults to none. */
+  @Parameter(property = "password")
+  private String password;
 
-    /**
-     * Connect to the database with this user id.
-     */
-    @Parameter(property = "user")
-    private String user;
+  /**
+   * If specified, SchemaSpy will look for JDBC drivers on this path, rather than using the
+   * application classpath. Useful if your database has a non-O/S driver not bundled with the
+   * plugin.
+   */
+  @Parameter(property = "pathToDrivers")
+  private String pathToDrivers;
 
-    /**
-     * Database schema to use - defaults to the specified user.
-     */
-    @Parameter(property = "schema")
-    private String schema;
+  /**
+   * Schema description. Displays the specified textual description on summary pages. If your
+   * description includes an equals sign then escape it with a backslash. NOTE: This field doesn't
+   * seem to be used by SchemaSpy in the current version.
+   */
+  @Parameter(property = "schemaDescription")
+  private String schemaDescription;
 
-    /**
-     * Database password to use - defaults to none.
-     */
-    @Parameter(property = "password")
-    private String password;
+  /**
+   * Only include matching tables/views. This is a regular expression that's used to determine which
+   * tables/views to include. For example: -i "(.*book.*)|(library.*)" includes only those
+   * tables/views with 'book' in their names or that start with 'library'. You might want to use
+   * "description" with this option to describe the subset of tables.
+   */
+  @Parameter(property = "includeTableNamesRegex")
+  private String includeTableNamesRegex;
 
-    /**
-     * If specified, SchemaSpy will look for JDBC drivers on this path, rather
-     * than using the application classpath. Useful if your database has a
-     * non-O/S driver not bundled with the plugin.
-     */
-    @Parameter(property = "pathToDrivers")
-    private String pathToDrivers;
+  /**
+   * Exclude matching columns from relationship analysis to simplify the generated graphs. This is a
+   * regular expression that's used to determine which columns to exclude. It must match table name,
+   * followed by a dot, followed by column name. For example: -x "(book.isbn)|(borrower.address)"
+   * Note that each column name regular expression must be surrounded by ()'s and separated from other
+   * column names by a |.
+   */
+  @Parameter(property = "excludeColumnNamesRegex")
+  private String excludeColumnNamesRegex;
 
-    /**
-     * Schema description. Displays the specified textual description on summary
-     * pages. If your description includes an equals sign then escape it with a
-     * backslash. NOTE: This field doesn't seem to be used by SchemaSpy in the
-     * current version.
-     */
-    @Parameter(property = "schemaDescription")
-    private String schemaDescription;
+  /**
+   * Allow HTML In Comments. Any HTML embedded in comments normally gets encoded so that it's
+   * rendered as text. This option allows it to be rendered as HTML.
+   */
+  @Parameter(property = "allowHtmlInComments")
+  private Boolean allowHtmlInComments;
 
-    /**
-     * Only include matching tables/views. This is a regular expression that's
-     * used to determine which tables/views to include. For example: -i
-     * "(.*book.*)|(library.*)" includes only those tables/views with 'book' in
-     * their names or that start with 'library'. You might want to use
-     * "description" with this option to describe the subset of tables.
-     */
-    @Parameter(property = "includeTableNamesRegex")
-    private String includeTableNamesRegex;
+  /**
+   * Comments Initially Displayed. Column comments are normally hidden by default. This option
+   * displays them by default.
+   *
+   * @deprecated this seems to no longer be a supported option in SchemaSpy
+   */
+  @Deprecated
+  @Parameter(property = "commentsInitiallyDisplayed")
+  private Boolean commentsInitiallyDisplayed;
 
-    /**
-     * Exclude matching columns from relationship analysis to simplify the
-     * generated graphs. This is a regular expression that's used to determine
-     * which columns to exclude. It must match table name, followed by a dot,
-     * followed by column name. For example: -x "(book.isbn)|(borrower.address)"
-     * Note that each column name regular expression must be surround by ()'s
-     * and separated from other column names by a |.
-     */
-    @Parameter(property = "excludeColumnNamesRegex")
-    private String excludeColumnNamesRegex;
+  /** Don't include implied foreign key relationships in the generated table details. */
+  @Parameter(property = "noImplied")
+  private Boolean noImplied;
 
-    /**
-     * Allow HTML In Comments. Any HTML embedded in comments normally gets
-     * encoded so that it's rendered as text. This option allows it to be
-     * rendered as HTML.
-     */
-    @Parameter(property = "allowHtmlInComments")
-    private Boolean allowHtmlInComments;
+  /** Only generate files needed for insertion/deletion of data (e.g. for scripts). */
+  @Parameter(property = "noHtml")
+  private Boolean noHtml;
 
-    /**
-     * Comments Initially Displayed. Column comments are normally hidden by
-     * default. This option displays them by default.
-     *
-     * @deprecated this seems to no longer be a supported option in SchemaSpy
-     */
-    @Deprecated
-    @Parameter(property = "commentsInitiallyDisplayed")
-    private Boolean commentsInitiallyDisplayed;
+  /** Detail of execution logging. */
+  @Parameter(property = "logLevel")
+  private String logLevel;
 
-    /**
-     * Don't include implied foreign key relationships in the generated table
-     * details.
-     */
-    @Parameter(property = "noImplied")
-    private Boolean noImplied;
+  /**
+   * Some databases, like Derby, will crash if you use the old driver object to establish a
+   * connection (eg "connection = driver.connect(...)"). In this case, set useDriverManager to true
+   * to use the DriverManager.getConnection() method instead (eg "connection =
+   * java.sql.DriverManager.getConnection(...)"). Other databases (e.g. MySQL) seem to only work with
+   * the first method, so don't use this parameter unless you have to.
+   */
+  @Parameter(property = "useDriverManager")
+  private Boolean useDriverManager;
 
-    /**
-     * Only generate files needed for insertion/deletion of data (e.g. for
-     * scripts).
-     */
-    @Parameter(property = "noHtml")
-    private Boolean noHtml;
+  /** The CSS Stylesheet. Allows you to override the default SchemaSpyCSS stylesheet. */
+  @Parameter(property = "cssStylesheet")
+  private String cssStylesheet;
 
-    /**
-     * Detail of execution logging.
-     */
-    @Parameter(property = "logLevel")
-    private String logLevel;
+  /**
+   * Single Sign-On. Don't require a user to be specified with -user to simplify configuration when
+   * running in a single sign-on environment.
+   */
+  @Parameter(property = "singleSignOn")
+  private Boolean singleSignOn;
 
-    /**
-     * Some databases, like Derby, will crash if you use the old driver object
-     * to establish a connection (eg "connection = driver.connect(...)"). In
-     * this case, set useDriverManager to true to use the
-     * DriverManager.getConnection() method instead (eg "connection =
-     * java.sql.DriverManager.getConnection(...)"). Other databases (eg MySQL)
-     * seem to only work with the first method, so don't use this parameter
-     * unless you have to.
-     */
-    @Parameter(property = "useDriverManager")
-    private Boolean useDriverManager;
+  /**
+   * Generate lower-quality diagrams. Various installations of Graphviz (depending on OS and/or
+   * version) will default to generating either higher or lower quality images. That is, some might
+   * not have the "lower quality" libraries and others might not have the "higher quality"
+   * libraries.
+   */
+  @Parameter(property = "lowQuality")
+  private Boolean lowQuality;
 
-    /**
-     * The CSS Stylesheet. Allows you to override the default SchemaSpyCSS
-     * stylesheet.
-     */
-    @Parameter(property = "cssStylesheet")
-    private String cssStylesheet;
+  /**
+   * Generate higher-quality diagrams. Various installations of Graphviz (depending on OS and/or
+   * version) will default to generating either higher or lower quality images. That is, some might
+   * not have the "lower quality" libraries and others might not have the "higher quality"
+   * libraries.
+   */
+  @Parameter(property = "highQuality")
+  private Boolean highQuality;
 
-    /**
-     * Single Sign-On. Don't require a user to be specified with -user to
-     * simplify configuration when running in a single sign-on environment.
-     */
-    @Parameter(property = "singleSignOn")
-    private Boolean singleSignOn;
+  /**
+   * Evaluate all schemas in a database. Generates a high-level index of the schemas evaluated and
+   * allows for traversal of cross-schema foreign key relationships. Use with -schemaSpec
+   * "schemaRegularExpression" to narrow-down the schemas to include.
+   */
+  @Parameter(property = "showAllSchemas")
+  private Boolean showAllSchemas;
 
-    /**
-     * Generate lower-quality diagrams. Various installations of Graphviz (depending on OS
-     * and/or version) will default to generating either higher or lower quality images.
-     * That is, some might not have the "lower quality" libraries and others might not have
-     * the "higher quality" libraries.
-     */
-    @Parameter(property = "lowQuality")
-    private Boolean lowQuality;
+  /**
+   * Evaluate specified schemas. Similar to -showAllSchemas, but explicitly specifies which schema
+   * to evaluate without interrogating the database's metadata. Can be used with databases like
+   * MySQL where a database isn't composed of multiple schemas.
+   */
+  @Parameter(property = "schemas")
+  private String schemas;
 
-    /**
-     * Generate higher-quality diagrams. Various installations of Graphviz (depending on OS
-     * and/or version) will default to generating either higher or lower quality images.
-     * That is, some might not have the "lower quality" libraries and others might not have
-     * the "higher quality" libraries.
-     */
-    @Parameter(property = "highQuality")
-    private Boolean highQuality;
+  /** No schema required for this database (e.g. derby). */
+  @Parameter(property = "noSchema")
+  private Boolean noSchema;
 
-    /**
-     * Evaluate all schemas in a database. Generates a high-level index of the schemas
-     * evaluated and allows for traversal of cross-schema foreign key relationships.
-     * Use with -schemaSpec "schemaRegularExpression" to narrow-down the schemas to include.
-     */
-    @Parameter(property = "showAllSchemas")
-    private Boolean showAllSchemas;
+  /** Don't query or display row counts. */
+  @Parameter(property = "noRows")
+  private Boolean noRows;
 
-    /**
-     * Evaluate specified schemas.
-     * Similar to -showAllSchemas, but explicitly specifies which schema to evaluate without
-     * interrogating the database's metadata. Can be used with databases like MySQL where a
-     * database isn't composed of multiple schemas.
-     */
-    @Parameter(property = "schemas")
-    private String schemas;
+  /** Don't query or display row counts. */
+  @Parameter(property = "noViews")
+  private Boolean noViews;
 
-    /**
-     * No schema required for this database (e.g. derby).
-     */
-    @Parameter(property = "noSchema")
-    private Boolean noSchema;
+  /**
+   * Specifies additional properties to be used when connecting to the database. Specify the entries
+   * directly, escaping the ='s with \= and separating each key\=value pair with a ;.
+   *
+   * <p>May also be a file name, useful for hiding connection properties from public logs.
+   */
+  @Parameter(property = "connprops")
+  private String connprops;
 
-    /**
-     * Don't query or display row counts.
-     */
-    @Parameter(property = "noRows")
-    private Boolean noRows;
+  /**
+   * Don't generate ads in reports.
+   *
+   * @deprecated will be removed
+   */
+  @Deprecated
+  @Parameter(property = "noAds", defaultValue = "true")
+  private Boolean noAds;
 
-    /**
-     * Don't query or display row counts.
-     */
-    @Parameter(property = "noViews")
-    private Boolean noViews;
+  /**
+   * Don't generate sourceforge logos in reports.
+   *
+   * @deprecated will be removed
+   */
+  @Deprecated
+  @Parameter(property = "noLogo", defaultValue = "true")
+  private Boolean noLogo;
 
-    /**
-     * Specifies additional properties to be used when connecting to the database.
-     * Specify the entries directly, escaping the ='s with \= and separating each key\=value
-     * pair with a ;.
-     * 
-     * May also be a file name, useful for hiding connection properties from public logs. 
-     */
-    @Parameter(property = "connprops")
-    private String connprops;
+  @Parameter(property = "catalog", defaultValue = "%")
+  private String catalog;
 
-    /**
-     * Don't generate ads in reports.
-     *
-     * @deprecated will be removed
-     */
-    @Deprecated
-    @Parameter(property = "noAds", defaultValue = "true")
-    private Boolean noAds;
+  /**
+   * The SchemaSpy analyser that generates the actual report. Can be overridden for testing
+   * purposes.
+   */
+  private MavenSchemaAnalyzer analyzer;
 
-    /**
-     * Don't generate sourceforge logos in reports.
-     *
-     * @deprecated will be removed
-     */
-    @Deprecated
-    @Parameter(property = "noLogo", defaultValue = "true")
-    private Boolean noLogo;
+  protected void setSchemaAnalyzer(final MavenSchemaAnalyzer analyzer) {
+    this.analyzer = analyzer;
+  }
 
-    @Parameter(property = "catalog", defaultValue = "%")
-    private String catalog;
-
-    @Parameter(property = "vizjs", defaultValue = "true")
-    protected boolean vizjs = false;
-
-    /**
-     * Whether to create the report only on the execution root of a multi-module project.
-     *
-     * @since 5.0.4
-     */
-    @Parameter(property = "runOnExecutionRoot", defaultValue = "false")
-    protected boolean runOnExecutionRoot = false;
-
-    /**
-     * The SchemaSpy analyser that generates the actual report.
-     * Can be overridden for testing purposes.
-     */
-    private MavenSchemaAnalyzer analyzer;
-
-    protected void setSchemaAnalyzer(final MavenSchemaAnalyzer analyzer) {
-        this.analyzer = analyzer;
+  /**
+   * Convenience method used to build the schemaspy command line parameters.
+   *
+   * @param argList the current list of schemaspy parameter options.
+   * @param parameter a new parameter to add
+   * @param value the value for this parameter
+   */
+  private void addToArguments(
+      final List<String> argList, final String parameter, final Boolean value) {
+    if (value != null && value) {
+      argList.add(parameter);
+      argList.add(String.valueOf(true));
     }
+  }
 
-    /**
-    * Convenience method used to build the schemaspy command line parameters.
-    *
-    * @param argList the current list of schemaspy parameter options.
-    * @param parameter a new parameter to add
-    * @param value the value for this parameter
-    */
-    private void addToArguments(
-        final List<String> argList, final String parameter, final Boolean value) {
-        if (value != null && value) {
-            argList.add(parameter);
-            argList.add(String.valueOf(true));
-        }
+  /**
+   * Convenience method used to build the schemaspy command line parameters.
+   *
+   * @param argList the current list of schemaspy parameter options.
+   * @param parameter a new parameter to add
+   * @param value the value for this parameter
+   */
+  private void addFlagToArguments(
+      final List<String> argList, final String parameter, final Boolean value) {
+    if (value != null && value) {
+      argList.add(parameter);
     }
+  }
 
-    /**
-    * Convenience method used to build the schemaspy command line parameters.
-    *
-    * @param argList the current list of schemaspy parameter options.
-    * @param parameter a new parameter to add
-    * @param value the value for this parameter
-    */
-    private void addFlagToArguments(
-        final List<String> argList, final String parameter, final Boolean value) {
-        if (value != null && value) {
-            argList.add(parameter);
-        }
+  /**
+   * Convenience method used to build the schemaspy command line parameters.
+   *
+   * @param argList the current list of schemaspy parameter options.
+   * @param parameter a new parameter to add
+   * @param value the value for this parameter
+   */
+  private void addToArguments(
+      final List<String> argList, final String parameter, final String value) {
+    if (value != null) {
+      argList.add(parameter);
+      argList.add(value);
     }
+  }
 
-    /**
-     * Convenience method used to build the schemaspy command line parameters.
-     *
-     * @param argList
-     *            the current list of schemaspy parameter options.
-     * @param parameter
-     *            a new parameter to add
-     * @param value
-     *            the value for this parameter
-     */
-    private void addToArguments(final List<String> argList, final String parameter, final String value) {
-        if (value != null) {
-            argList.add(parameter);
-            argList.add(value);
-        }
+  /**
+   * Generate the Schemaspy report.
+   *
+   * @throws MavenReportException if schemaspy crashes
+   * @param locale the language of the report — currently ignored.
+   */
+  @Override
+  protected void executeReport(final Locale locale) throws MavenReportException {
+    File outputDir;
+    if (outputDirectory == null) {
+      // targetDirectory should be set by the maven framework. This is just for unit testing
+      // purposes
+      if (targetDirectory == null) {
+        targetDirectory = new File("target");
+      }
+      File siteDir = new File(targetDirectory, "site");
+      outputDir = new File(siteDir, "schemaspy");
+    } else {
+      outputDir = new File(outputDirectory, "schemaspy");
     }
+    outputDir.mkdirs();
 
-    /**
-     * Generate the Schemaspy report.
-     *
-     * @throws MavenReportException
-     *             if schemaspy crashes
-     * @param locale
-     *            the language of the report - currently ignored.
-     */
-    @Override
-    protected void executeReport(final Locale locale) throws MavenReportException {
+    String schemaSpyDirectory = outputDir.getAbsolutePath();
+    getLog().debug("SchemaSpy output directory: " + schemaSpyDirectory);
 
-        //
-        // targetDirectory should be set by the maven framework. This is
-        // jusr for unit testing purposes
-        //
-        if (targetDirectory == null) {
-            targetDirectory = new File("target");
-        }
+    List<String> argList = new ArrayList<>();
 
-        targetDirectory.mkdirs();
-        File siteDir = new File(targetDirectory, "site");
-        siteDir.mkdirs();
-        File outputDir = null;
-        if (outputDirectory == null) {
-            outputDir = new File(siteDir, "schemaspy");
-            outputDir.mkdirs();
-            outputDirectory = outputDir.getAbsolutePath();
-        } else {
-            outputDir = new File(new File(outputDirectory), "schemaspy");
-            outputDir.mkdirs();
-        }
-        String schemaSpyDirectory = outputDir.getAbsolutePath();
-        getLog().debug("SchemaSpy output directory: " + schemaSpyDirectory);
-
-        List<String> argList = new ArrayList<>();
-
-//        if ((jdbcUrl != null) && (databaseType == null)) {
-//            databaseType = JDBCHelper.extractDatabaseType(jdbcUrl);
-//        }
-        addToArguments(argList, "-dp", pathToDrivers);
-        addToArguments(argList, "-db", database);
-        addToArguments(argList, "-host", host);
-        addToArguments(argList, "-port", port);
-        addToArguments(argList, "-t", databaseType);
-        addToArguments(argList, "-u", user);
-        addToArguments(argList, "-p", password);
-        if (null != schema && schema.contains(",")) {
-            addToArguments(argList, "-schemas", schema);
-        } else {
-            addToArguments(argList, "-s", schema);
-        }
-        addToArguments(argList, "-o", schemaSpyDirectory);
-        addToArguments(argList, "-desc", schemaDescription);
-        addToArguments(argList, "-i", includeTableNamesRegex);
-        addToArguments(argList, "-x", excludeColumnNamesRegex);
-        addFlagToArguments(argList, "-ahic", allowHtmlInComments);
-        addFlagToArguments(argList, "-noimplied", noImplied);
-        addFlagToArguments(argList, "-nohtml", noHtml);
-        addToArguments(argList, "-loglevel", logLevel);
-        addFlagToArguments(argList, "-norows", noRows);
-        addFlagToArguments(argList, "-noviews", noViews);
-        addFlagToArguments(argList, "-noschema", noSchema);
-        addFlagToArguments(argList, "-all", showAllSchemas);
-        addToArguments(argList, "-schemas", schemas);
-
-        addToArguments(argList, "-useDriverManager", useDriverManager);
-        addToArguments(argList, "-css", cssStylesheet);
-        addFlagToArguments(argList, "-sso", singleSignOn);
-        addFlagToArguments(argList, "-lq", lowQuality);
-        addFlagToArguments(argList, "-hq", highQuality);
-        addToArguments(argList, "-connprops", connprops);
-        addFlagToArguments(argList, "-cid", commentsInitiallyDisplayed);
-        addFlagToArguments(argList, "-noads", noAds);
-        addFlagToArguments(argList, "-nologo", noLogo);
-        addToArguments(argList, "-cat", catalog);
-        addFlagToArguments(argList, "-vizjs", vizjs);
-        if(getLog().isDebugEnabled()) {
-            addFlagToArguments(argList, "-debug", true);
-        }
-//        addToArguments(argList, "-jdbcUrl", jdbcUrl);
-        getLog().debug("SchemaSpy arguments: " + argList);
-
-
-        try {
-            if (analyzer == null) {
-                analyzer = new MavenSchemaAnalyzer();
-                analyzer.applyConfiguration(argList);
-            }
-            analyzer.analyze();
-        } catch (Exception e) {
-            throw new MavenReportException(e.getMessage(), e);
-        }
+    addToArguments(argList, "-dp", pathToDrivers);
+    addToArguments(argList, "-db", database);
+    addToArguments(argList, "-host", host);
+    addToArguments(argList, "-port", port);
+    addToArguments(argList, "-t", databaseType);
+    addToArguments(argList, "-u", user);
+    addToArguments(argList, "-p", password);
+    if (null != schema && schema.contains(",")) {
+      addToArguments(argList, "-schemas", schema);
+    } else {
+      addToArguments(argList, "-s", schema);
     }
-
-    @Override
-    public boolean canGenerateReport() {
-        return !runOnExecutionRoot || project.isExecutionRoot();
+    addToArguments(argList, "-o", schemaSpyDirectory);
+    addToArguments(argList, "-desc", schemaDescription);
+    addToArguments(argList, "-i", includeTableNamesRegex);
+    addToArguments(argList, "-x", excludeColumnNamesRegex);
+    addFlagToArguments(argList, "-ahic", allowHtmlInComments);
+    addFlagToArguments(argList, "-noimplied", noImplied);
+    addFlagToArguments(argList, "-nohtml", noHtml);
+    addToArguments(argList, "-loglevel", logLevel);
+    addFlagToArguments(argList, "-norows", noRows);
+    addFlagToArguments(argList, "-noviews", noViews);
+    addFlagToArguments(argList, "-noschema", noSchema);
+    addFlagToArguments(argList, "-all", showAllSchemas);
+    addToArguments(argList, "-schemas", schemas);
+    addToArguments(argList, "-useDriverManager", useDriverManager);
+    addToArguments(argList, "-css", cssStylesheet);
+    addFlagToArguments(argList, "-sso", singleSignOn);
+    addFlagToArguments(argList, "-lq", lowQuality);
+    addFlagToArguments(argList, "-hq", highQuality);
+    addToArguments(argList, "-connprops", connprops);
+    addFlagToArguments(argList, "-cid", commentsInitiallyDisplayed);
+    addFlagToArguments(argList, "-noads", noAds);
+    addFlagToArguments(argList, "-nologo", noLogo);
+    addToArguments(argList, "-cat", catalog);
+    addFlagToArguments(argList, "-vizjs", vizjs);
+    if (getLog().isDebugEnabled()) {
+      addFlagToArguments(argList, "-debug", true);
     }
+    getLog().debug("SchemaSpy arguments: " + argList);
 
-    @Override
-    protected String getOutputDirectory() {
-        return outputDirectory;
+    try {
+      if (analyzer == null) {
+        analyzer = new MavenSchemaAnalyzer();
+        analyzer.applyConfiguration(argList);
+      }
+      analyzer.analyze();
+    } catch (Exception e) {
+      throw new MavenReportException(e.getMessage(), e);
     }
+  }
 
-    @Override
-    protected MavenProject getProject() {
-        return project;
-    }
+  @Override
+  public boolean canGenerateReport() {
+    return !runOnExecutionRoot || project.isExecutionRoot();
+  }
 
-    @Override
-    protected Renderer getSiteRenderer() {
-        return siteRenderer;
-    }
+  @Override
+  public String getDescription(final Locale locale) {
+    return "SchemaSpy database documentation";
+  }
 
-    @Override
-    public String getDescription(final Locale locale) {
-        return "SchemaSpy database documentation";
-    }
+  @Override
+  public String getName(final Locale locale) {
+    return "SchemaSpy";
+  }
 
-    @Override
-    public String getName(final Locale locale) {
-        return "SchemaSpy";
-    }
+  @Override
+  public String getOutputName() {
+    return "schemaspy/index";
+  }
 
-    @Override
-    public String getOutputName() {
-        return "schemaspy/index";
-    }
+  @Override
+  public String getOutputPath() {
+    return "schemaspy/index";
+  }
 
-    /**
-     * Always return {@code true} as we're using the report generated by SchemaSpy
-     * rather than creating our own report.
-     *
-     * @return {@code true}
-     */
-    @Override
-    public boolean isExternalReport() {
-        return true;
-    }
+  /**
+   * Always return {@code true} as we're using the report generated by SchemaSpy rather than
+   * creating our own report.
+   *
+   * @return {@code true}
+   */
+  @Override
+  public boolean isExternalReport() {
+    return true;
+  }
 }

--- a/src/test/java/com/wakaleo/schemaspy/CommandlineIntegrationTest.java
+++ b/src/test/java/com/wakaleo/schemaspy/CommandlineIntegrationTest.java
@@ -1,0 +1,105 @@
+/*
+ * Copyright 2025 Mark Prins, GeoDienstenCentrum.
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its contributors
+ * may be used to endorse or promote products derived from this software without
+ * specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+ * OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.wakaleo.schemaspy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.nio.file.Paths;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+
+/** Testcases for the {@code default-cli} commandline. */
+public class CommandlineIntegrationTest {
+  private static final String pomFile = "src/test/projects/unit/pgsql-plugin-config-cli.xml";
+  @Rule public TestName testName = new TestName();
+  private String version;
+
+  @Before
+  public void setUp() {
+    version = System.getenv("PLUGIN_VERSION");
+  }
+
+  @Test
+  public void testCommandlineHelp() throws Exception {
+    Process mvn =
+        new ProcessBuilder(
+                "mvn",
+                "schemaspy:help",
+                // "-X",
+                "-U",
+                "-V",
+                "-f",
+                pomFile,
+                "-Dplugin.version=" + version)
+            .redirectErrorStream(true)
+            .start();
+
+    BufferedReader stdOutReader = new BufferedReader(new InputStreamReader(mvn.getInputStream()));
+    String line;
+    while ((line = stdOutReader.readLine()) != null) {
+      System.out.println(testName.getMethodName() + "--" + line);
+    }
+    assertEquals("Maven default-cli 'schemaspy:help' command failed", 0, mvn.waitFor());
+  }
+
+  @Test
+  public void testCommandlineSchemaspy() throws Exception {
+    // /target/reports/pgsql-cli-test/schemaspy/index.html
+    final String generatedFile = "./target/reports/pgsql-cli-test/schemaspy/index.html";
+    Process mvn =
+        new ProcessBuilder(
+                "mvn",
+                "schemaspy:schemaspy",
+                // "-X",
+                "-U",
+                "-V",
+                "-f",
+                pomFile,
+                // "-DoutputDirectory=./../../../../target/reports/pgsql-cli-test"
+                "-Dplugin.version=" + version)
+            .redirectErrorStream(true)
+            .start();
+
+    BufferedReader stdOutReader = new BufferedReader(new InputStreamReader(mvn.getInputStream()));
+    String line;
+    while ((line = stdOutReader.readLine()) != null) {
+      System.out.println(testName.getMethodName() + "--" + line);
+    }
+
+    assertEquals("Maven default-cli 'schemaspy:schemaspy' command failed", 0, mvn.waitFor());
+    assertTrue(Paths.get(generatedFile).toFile().exists());
+  }
+}

--- a/src/test/projects/unit/pgsql-plugin-config-cli.xml
+++ b/src/test/projects/unit/pgsql-plugin-config-cli.xml
@@ -1,0 +1,71 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>custom.configuration</groupId>
+  <artifactId>custom-configuration</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <inceptionYear>2018</inceptionYear>
+  <name>Maven SchemaSpy Plugin Configuration Test</name>
+  <url>http://maven.apache.org</url>
+    <properties>
+        <plugin.version/>
+    </properties>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>as-build</id>
+            <url>file://${project.basedir}/../../../../target/testRepo</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </pluginRepository>
+    </pluginRepositories>
+    <repositories>
+        <repository>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+            <id>as-build</id>
+            <url>file://${project.basedir}/../../../../target/testRepo</url>
+        </repository>
+    </repositories>
+  <build>
+    <finalName>pgsql-test-plugin-configuration</finalName>
+    <plugins>
+        <plugin>
+            <groupId>nl.geodienstencentrum.maven</groupId>
+            <artifactId>schemaspy-maven-plugin</artifactId>
+            <version>${plugin.version}</version>
+            <configuration>
+              <databaseType>pgsql</databaseType>
+              <database>test</database>
+              <schema>public</schema>
+              <user>postgres</user>
+              <password>postgres</password>
+              <host>127.0.0.1</host>
+              <port>5432</port>
+              <outputDirectory>${project.basedir}/../../../../target/reports/pgsql-cli-test</outputDirectory>
+              <schemaDescription>Maven Plugin SchemaSpy test</schemaDescription>
+            </configuration>
+        </plugin> 
+    </plugins>
+  </build>
+</project>

